idtype/idtype.go

```diff
@@ -117,7 +117,7 @@ func (t Type) Collection() string {
 	case strings.HasSuffix(defn.name, "access"):
```

Comment:
Could we not just add an exception here?

Reply:
I'm not sure what this is, I'd prefer to not touch it 😅

Reply:
I mean: `if drf.name == "category" return "categories"` :)

Reply:
@jelmersnoeck suggested using using a plural method and making it extensible, this case might happen again and it'd be great to not have to hack this every time.